app/Nova/Travel.php

```diff
@@ -208,6 +208,22 @@ public function fields(Request $request): array
                             ' If this is not applicable, enter 0.'
                         )
                         ->hideFromIndex(),
+
+                    Text::make('Workday Project Number', 'tar_project_number')
```

Comment:
Would this make more sense as a dropdown?

Reply:
I feel like a dropdown would somewhat discourage getting the real answer from GT.

Reply:
Makes sense.